### ISSUE TITLE
Holstein Friesian Giving Birth To Black Angus

### ISSUE THREAD

**Issue by Animalover609** [nfinit-gaming/BetterAgriculture#60]
My Holstein Friesian cows are only giving birth to Black Angus'. Currently, I don't think there is a way to maintain my Holstein Friesian flock if I use them for food too. Maybe I'm misunderstanding how cow breeding worlds irl, but is this correct or is it a bug?? Thank you.

**Comment by Faenum** [comment]
no, thats a bug, thanks for finding and reporting it, will be fixed in the next version
